backend/models/user.ts

```
@@ -161,19 +183,49 @@
+      this.projects.assigned.push(p)
     }
   }
+
+  for (const p of userToOverwrite.projects!.supervised) {
+    if (!this.projects.supervised.some((tp) => tp._id.equals(p._id))) {
+      this.projects.supervised.push(p)
+    }
+  }
+
+  Object.assign(this.settings, userToOverwrite.settings, thisPojo.settings)
   delete userToOverwrite.settings
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)